### DISCUSSION TITLE
Default collection sort on CP?

### DISCUSSION THREAD

**Discussion by david-windsock** [statamic/cms#9249] [Q&A] [answered]
Is there any way to save a default sort for collections on control panel?. I know we can save the columns and its position but didn't find anything to apply a default sort. Can this be set on any YAML file?

I don't want to make the collection orderable, just set a specific sorting on its index on control panel...

Thanks!

**Comment by ryanmitchell** [comment, marked as answer]
You can add `sort_by` to your YAML (along with `sort_dir`)
eg `sort_by: my_field_handle`

**Comment by david-windsock** [comment]
Thanks @ryanmitchell, it worked. I see now that I missed that on docs: https://statamic.dev/collections#default-sort-order-in-listings
